Formula/libfido2.rb

```diff
@@ -0,0 +1,46 @@
+class Libfido2 < Formula
+  desc "Provides library functionality for FIDO U2F & FIDO 2.0, including USB"
+  homepage "https://developers.yubico.com/libfido2/"
+  url "https://github.com/Yubico/libfido2/archive/1.3.0.tar.gz"
+  sha256 "0b2e3671c4c5d42fd5604a08e45f89f49592b97cf66d7d3bfbc7e6a4d5a0fec7"
+  depends_on "cmake" => :build
+  depends_on "mandoc" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libcbor"
+  depends_on "openssl@1.1"
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
```

Comment:
These regression tests should get run in the the previous step, but you could explicitly call for them with this change:
```suggestion
      system "make"
      system "make", "regress"
```

I wouldn't bother changing anything else. I'm happy with this as is, or if you want to accept this change.

Reply:
I tried `make regress` once more, but it failed: (I now recall that this was why I had not used it in the first place.)
```make: *** No rule to make target `regress'.  Stop.```
As you've said, I think we'll be fine without it.

Thank you for the thorough review and lending your CMake knowledge to this PR!